content/pages/health-care/about-va-health-care/va-health-care-and-other-insurance.md

```diff
@@ -5,7 +5,7 @@ title: VA Health Care and Other Insurance
 display_title:
 concurrence: complete
 lastupdate: 2017-06-28
-order: 10
+order: 3
```

Comment:
I just used what was in `master`, hoping this was right. Not sure who to ping on this one.